mediator/src/main/scala/io/iohk/atala/mediator/actions/ActionUtils.scala

```diff
@@ -1,158 +1,157 @@
-package io.iohk.atala.mediator.actions
+// package io.iohk.atala.mediator.actions
 
-import fmgp.crypto.error.*
-import fmgp.did.*
-import fmgp.did.comm.*
-import fmgp.did.comm.Operations.*
-import fmgp.did.comm.protocol.*
-import fmgp.did.comm.protocol.basicmessage2.*
-import fmgp.did.comm.protocol.trustping2.*
-import io.iohk.atala.mediator.*
-import io.iohk.atala.mediator.comm.*
-import io.iohk.atala.mediator.db.*
-import zio.*
-import zio.json.*
-import io.iohk.atala.mediator.protocols.MissingProtocolExecuter
+// import zio.*
+// import zio.json.*
 
-object ActionUtils {
+// import fmgp.crypto.error.*
+// import fmgp.did.*
+// import fmgp.did.comm.*
+// import fmgp.did.comm.Operations.*
+// import fmgp.did.comm.protocol.*
+// import io.iohk.atala.mediator.*
+// import io.iohk.atala.mediator.comm.*
+// import io.iohk.atala.mediator.db.*
 
-  def packResponse(
-      originalMessage: Option[PlaintextMessage],
-      action: Action
-  ): ZIO[
-    Operations & Agent & Resolver & MessageDispatcher & OutboxMessageRepo,
-    MediatorError,
-    Option[SignedMessage | EncryptedMessage]
-  ] =
-    action match {
-      case _: NoReply.type => ZIO.succeed(None)
-      case action: AnyReply =>
-        val reply = action.msg
-        for {
+// object ActionUtils {
```

Comment:
Do we still need this ?

Reply:
I think I still need parts of this code.
But I need to run the test suite first. I don't know why the test suite is green...